apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/BigDecimal.kt

```diff
@@ -0,0 +1,7 @@
+package com.apollographql.apollo.api
+
+expect class BigDecimal(value: String?) {
```

Comment:
I have in one of my project this https://gist.github.com/sav007/8bf365e7eaa86f4f4b6c890ea3ce47e6